### DISCUSSION TITLE
MacOS app icon does not match apple HIG

### DISCUSSION THREAD

**Discussion by walsvid** [massCodeIO/massCode#121] [Ideas]
The current macos icon does not meet the requirements of [human-interface-guidelines](https://developer.apple.com/design/human-interface-guidelines/foundations/app-icons#platform-considerations), and the icon format is not good enough on macos11 and above systems.

FYI, I update the icon according to HIG. Are we considering using HIG-compliant icons on macOS?

[massCode.icns.zip](https://github.com/massCodeIO/massCode/files/9081048/massCode.icns.zip)
<img src="https://user-images.githubusercontent.com/17134911/178192748-a16cc5c2-306b-4038-a438-dc298647978f.jpg" height=300px/>

**Comment by antonreshetov** [comment]
There are many apps that go their own way in this regard. I am not yet considering changing the icon of the application.

**Reply by walsvid** [reply]
OK, that's fine. I totally understand your decision.
